docs/source/en/model_doc/auto.mdx

```diff
@@ -74,33 +74,41 @@ Likewise, if your `NewModel` is a subclass of [`PreTrainedModel`], make sure its
 
 [[autodoc]] AutoProcessor
 
-## AutoModel
+## Generic model classes
+
+The following `AutoClasses` are available for instantiating a base model class without a specific head.
```

Comment:
Would remove the code block here as AutoClasses is not an object that exists in the lib. And I would spell it auto classes.